docs/basics.txt

```diff
@@ -333,16 +333,20 @@ However, note that this would not affect vertices, edges, or properties previous
 [[gremlin]]
 == Gremlin Query Language
 
-image:http://tinkerpop.apache.org/docs/{tinkerpop_version}/images/gremlin-logo.png[link="http://tinkerpop.apache.org/docs/{tinkerpop_version}/reference"]
+image:http://tinkerpop.apache.org/docs/{tinkerpop_version}/images/gremlin-logo.png[width=370,height=143,link="http://tinkerpop.apache.org/docs/{tinkerpop_version}/reference"]
```

Comment:
This image isn't linkable in the generated docs but it doesn't look like it is originally either.